### ISSUE TITLE
Multiple offline maps in a survey

### ISSUE THREAD

**Issue by stacylorraineprendeville** [FundacionParaguaya/MentorApp#619]
As a HUB user I want to administer one identical survey in three cities...

If user is offline and has multiple offline maps 
available show page with options available of offline maps 

![Screen Shot 2019-05-15 at 11.27.29.png](https://images.zenhubusercontent.com/5c19ffa9a896da5d83d6a0d0/4c10fa95-4128-4587-ad74-f7b8527eee9b)

if a user clicks on the link that none of those locations are relevant - then they are led to the location page.

**Comment by Danman7** [comment]
There is a reason for that. If we zoom out more than we have cached for offline use, the map might become gray. We will however initially at least allow better zoom out for online, and we can circumvent the offline problem as it is related to this #524.

**Comment by Danman7** [comment]
First we need the backend guys to allow us to add offline maps based on surveys. We are hardcoding them now. We would then cache all maps coming from the server and they will be available to anybody using the survey.

From there we would need a select UI to choose which portion of the map we want to center on, as well as to decide which will be the default one. This will be determined by the way they are sent from the server.

**Comment by stacylorraineprendeville** [comment]
This is the update from GECO: 

Las ciudades, con las respectivas fechas de visita son: Antofagasta y Mejillones el lunes 27 de mayo, Arica el lunes 2 de junio e Iquique el jueves 6 de junio.

Tenemos que ver si vamos a ser capaces de llegar a ello....

**Comment by Danman7** [comment]
For downloading maps I only really need the boundaries, I don't really need the zoom. We would also need a center for each map. They can be formulated like this:
```
{
  "surveyConfig": {
    "offlineMaps": [
      {
        "from": [45,34],
        "to": [21,34],
        "center": [22, 31]
      },
      {
        "from": [32,34],
        "to": [21,65],
        "center": [43, 31]
      }
    ]
  }
}
```

**Comment by stacylorraineprendeville** [comment]
@diegotorrespy @Danman7 @javierpf Can you let me know which server tasks are required - not clear...

**Comment by Danman7** [comment]
This is currently blocked by the backend.

@diegotorrespy Do you have an issue as I couldn't find one?

**Comment by Danman7** [comment]
Implemented in #708. The have the maps now.
